process/track/sovereignChainBlockProcessor.go

```diff
@@ -117,3 +118,11 @@ func (scbp *sovereignChainBlockProcessor) requestHeadersIfNothingNewIsReceived(
 
 	scbp.baseRequestHeadersIfNothingNewIsReceived(lastNotarizedHeaderNonce, latestValidHeader, highestRoundInReceivedHeaders, shardID)
 }
+
+func (scbp *sovereignChainBlockProcessor) getBlockFinality(shardID uint32) uint64 {
+	if shardID == core.MainChainShardId {
+		return 0
+	}
+
+	return scbp.blockFinality
```

Comment:
we should have instant finality here as well.

Reply:
Right, will simply change the variable to 0 then